### DISCUSSION TITLE
can't find zpool.cache file but pool imoorts fine?

### DISCUSSION THREAD

**Discussion by kocoman1** [openzfs/zfs#11961] [Q&A]
can't find zpool.cache file but pool imoorts fine?

can't find zpool.cache file but pool imoorts fine?

I am using 2.0.3 in lubuntu 20 (ubuntu) self complied openzfs

i run zdb -S tank but it cant find my pool tank

is there any verbose zpool import tank command to find where the cache is? 

i did a locate (after running updatedb) for zpool.cache but it does not exist

i look at the etc/default/zfs file and cant find anything special

i ran the "get all" on my pool and it show my cachefile is "-"

how do i create this binary cache file manually?

thx

**Comment by AllKind** [comment]
As you manually compiled it, a configure.log should have been created. It should tell you the "prefix" you used.
Most likely it is /usr/local/... So your cache file could be in /usr/local/etc/zfs. 
`systemctl show zfs-import-cache.service` will show you the path where it is trying to import the cache file (if you use systemd).
Also check if the service is enabled (`systemctl status zfs-import-cache.service`).

However helping yourself:
`man zpool-set`
and
`man zpoolprops`

Will show you with `zpool set cachefile=path` a path can be manually configured.
